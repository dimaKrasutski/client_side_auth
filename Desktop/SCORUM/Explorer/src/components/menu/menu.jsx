import React from 'react';
import { FormattedMessage } from 'react-intl';
import { RouteLink } from '../../components/route-link/route-link';
import './menu.scss';

export const Menu = () => (
  <div className="header-menu font-weight-bold text-small text-uppercase d-inline-block">
    <RouteLink to="/" className="mr-3_5 ml-5">
      <FormattedMessage id="explorer.menu.dashboard" />
    </RouteLink>
    <RouteLink to="/blocks" className="mr-3_5">
      <FormattedMessage id="explorer.menu.blocks" />
    </RouteLink>
    <RouteLink to="/transactions" className="mr-3_5">
      <FormattedMessage id="explorer.menu.transactions" />
    </RouteLink>
    <RouteLink to="/delegates">
      <FormattedMessage id="explorer.menu.delegates" />
    </RouteLink>
  </div>
);
