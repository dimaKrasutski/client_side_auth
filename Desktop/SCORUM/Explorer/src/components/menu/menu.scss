.header-menu {
  & > a {
    color: $white;
  }

  & > a:hover {
    text-decoration: none;
    color: rgba(255, 255, 255, 0.7);
  }
}
