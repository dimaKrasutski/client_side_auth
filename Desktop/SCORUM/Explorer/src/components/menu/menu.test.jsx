import React from 'react';
import { Menu } from './menu';

describe('<Menu />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<Menu />);
    expect(wrapper).toMatchSnapshot();
  });
});
