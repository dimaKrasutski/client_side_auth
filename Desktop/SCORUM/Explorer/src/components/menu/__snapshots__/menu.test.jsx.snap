// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> should match snapshot 1`] = `
<div
  className="header-menu font-weight-bold text-small text-uppercase d-inline-block"
>
  <Connect(RouteLink)
    className="mr-3_5 ml-5"
    to="/"
  >
    <FormattedMessage
      id="explorer.menu.dashboard"
      values={Object {}}
    />
  </Connect(RouteLink)>
  <Connect(RouteLink)
    className="mr-3_5"
    to="/blocks"
  >
    <FormattedMessage
      id="explorer.menu.blocks"
      values={Object {}}
    />
  </Connect(RouteLink)>
  <Connect(RouteLink)
    className="mr-3_5"
    to="/transactions"
  >
    <FormattedMessage
      id="explorer.menu.transactions"
      values={Object {}}
    />
  </Connect(RouteLink)>
  <Connect(RouteLink)
    to="/delegates"
  >
    <FormattedMessage
      id="explorer.menu.delegates"
      values={Object {}}
    />
  </Connect(RouteLink)>
</div>
`;
